client/src/modules/stock/requisition/modals/search.modal.html

```diff
@@ -38,6 +38,20 @@
             </div>
           </div>
 
+          <div>
+            <label class="control-label" translate>FORM.LABELS.STATUS</label>
+            <div class="row">
+              <div ng-repeat ="r in $ctrl.requisitionStatus | orderBy:'plainText'" class="col-md-6">
+                <div class="checkbox">
```

Comment:
This is a minor, cosmetic suggestion:  You could reduce the vertical space between the status check boxes to reduce the excess space that this selection takes.  Even better, if it could go into three columns that would help even more. Here is a suggestion for addressing this (notice the two "style" additions).
>           <div style="margin-bottom: 10px">
>             <label class="control-label" translate>FORM.LABELS.STATUS</label>
>             <div class="row">
>               <div ng-repeat ="r in $ctrl.requisitionStatus | orderBy:'plainText'" class="col-md-6">
>                 <div class="checkbox" style="margin-top: 0; margin-bottom: 2px">
>                   <label>
>                     <input type="checkbox" ng-model="r.checked" ng-true-value="1" ng-false-value="0">
>                     <span ng-class="r.class_style" translate> {{ r.plainText }} </span>
>                   </label>
>                 </div>
>               </div>
>             </div>
>           </div>

It seems inconvenient to have to scroll down to see the last field and the Submit/Cancel buttons.  Is it possible to make the search modal taller when it comes up to avoid scrolling?

Reply:
Very nice 
![image](https://user-images.githubusercontent.com/8035588/122919697-70eb8c80-d358-11eb-8db4-f6cf845fe77c.png)

Reply:
Looks good.